src/ceaser_cipher.js

```diff
@@ -0,0 +1,10 @@
+const cipherWord = (word, num) => word.split('').map(char => (char === char.toUpperCase()
+  ? String.fromCharCode(65 + ((char.charCodeAt() - 65 + num) % 26))
+  : String.fromCharCode(97 + ((char.charCodeAt() - 97 + num) % 26)))).join('');
+
+const cipher = (str, num) => {
+  const arr = str.split(' ');
+  return arr.map(word => cipherWord(word, num)).join(' ');
+};
+
+export default cipher;
```

Comment:
Special characters or punctuations should be preserved and not , encrypted
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/caesar_requirement.png)